main.html

```diff
@@ -0,0 +1,151 @@
+                            <img class="logo__image--dropdown" src="Logo.png" alt="Save the planet Logo">
+                        </a>
+                    </div>
+                    <li class="nav__item--home">
+                        <a class="nav__link--home" href="#">Home</a>
+                    </li>
+                    <li class="nav__item">
+                        <a class="nav__link--dropdown" href="#">Story</a>
+                    </li>
+                    <li class="nav__item--contact">
+                        <a class="nav__link--contact" href="#">Contact</a>
+                    </li>
+                    <li class="nav__item">
+                        <a class="nav__link--lessons" href="#">Lessons</a>
+                    </li>
+                    <li class="nav__item">
+                        <a class="nav__link--account" href="#">Account</a>
+                    </li>
+                </ul>
+                <div class="dropdown">
+                    <ul class="dropdown__list">
+                        <li class="dropdown__item">
+                            <a class="dropdown__link" href="#">Team</a>
+                        </li>
+
+                        <li class="dropdown__item">
+                            <a class="dropdown__link" href="#">Project</a>
+                        </li>
+
+                        <li class="dropdown__item">
+                            <a class="dropdown__link" href="#">Future</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <section class="hero">
+        <div class="container">
+            <div class="aside">
+                <aside class="aside__contact">
+                    <ul class="aside__content">
+                        <li class="aside__item">+04 070 991 598</li>
+                        <li class="aside__item">wild@life.com</li>
+                        <li class="aside__item">Wild Avenue, San Francisco, CA</li>
+                    </ul>
+                </aside>
+            </div>
+            <div class="hero__inner">
+                <p class="hero__heading">we teach you to protect our </p>
+                <h1 class="hero__title">planet</h1>
+                <p class="hero__content">Our planet is precious and vital for our beautiful lives,if we don't take care
+                    of it it will dissapear along with us.</p>
+                <p class="hero__undernote">*Please take note that your actions matter more than you think.</p>
+                <div class="hero__cta">
+                    <a class="hero__button" href="#">Find out more</a>
+                    <a class="hero__button--transparent" href="#">Play video</a>
+                </div>
+            </div>
+            <div class="hero__slider">
+                <div class="hero__sliderBtn" data-background = "hero-image.jpg"> 
+                    <p class="sliderBtn__caption">1</p> 

```

Comment:
aici nu e ok numele clasei, pune-ti un parinte .slider si mergi in jos cu slider__button, slider__caption, etc.